pkg/vpcmodel/mdOutput.go

```diff
@@ -50,6 +75,16 @@ func (m *MDoutputFormatter) getGroupedOutput(conn *VPCConnectivity) []string {
 	return lines
 }
 
+func (m *MDoutputFormatter) getGroupedDiffOutput(diff *diffBetweenCfgs) []string {
+	lines := make([]string, len(diff.groupedLines))
+	for i, line := range diff.groupedLines {
+		decodedDetails := strings.Split(line.Conn, semicolon)
+		lines[i] = fmt.Sprintf("| %s | %s | %s | %s | %s | %s |", decodedDetails[0], line.Src.Name(),
+			line.Dst.Name(), decodedDetails[1], decodedDetails[2], decodedDetails[3])
+	}
+	return lines
+}
```

Comment:
- missing documentation of expected format (an example in a comment could help), for which decoding relies on
- instead of decoding and relying on string formats, can use the added objects to `groupedLines`?

Reply:
already changed in merge with latest main